src/activity_type.cpp

```diff
@@ -32,6 +32,14 @@ static const std::unordered_map< std::string, based_on_type > based_on_type_valu
     { "neither", based_on_type::NEITHER }
 };
 
+static const std::map<std::string, float> activity_levels = {
+    { "NO_EXERCISE", NO_EXERCISE },
+    { "LIGHT_EXERCISE", LIGHT_EXERCISE },
+    { "MODERATE_EXERCISE", MODERATE_EXERCISE },
+    { "ACTIVE_EXERCISE", ACTIVE_EXERCISE },
+    { "EXTRA_EXERCISE", EXTRA_EXERCISE }
```

Comment:
Should probably clarify these for future contributors with a comment linking to details/definitions for each exercise level (making a comment of all of them here would take up a significant space).

The comment may be more appropriate at src/game_constants.h#L152 or maybe both places. Your choice.